trainer.py

```diff
@@ -33,7 +33,7 @@ def select_action(self, state):
         m = torch.distributions.Categorical(probs)
         action = m.sample()
         self.policy.saved_log_probs.append(m.log_prob(action))
-        return Actions[action.item()]
+        return Actions()[action.item()]
```

Comment:
If we are not going to use latest python, we can then just remove the whole actions class. Just leave the variable and consume it as normal array. It had some sense if `[i]` was an static statement. This feels a little bit overengineered.
Sounds a bit odd to instanciate a class for accessing to an static variable.

The `/spec` folder can be removed too because there's no test to apply to it.

wdyt?

Reply:
Honestly I'm not sure. IMO compatibility should be prioritised; if the class has any advantages over a regular array then I'm cool with it, but if there are no plans to do any fancy methods or reuse it in regular OOP fashion then sure, we can remove the class and declare Actions as a constant.